### PR TITLE
Add track-selection setters for GLO QC

### PR DESCRIPTION
**Pull request by mfaggin**
- proper setXXX() functions implemented in track-cut class and ITS-TPC matching efficiency task
 - separate cuts for ITS-TPC matched tracks from ITS and TPC cuts (isBarrel member removed)

### CODE REVIEW

**Comment by mfaggin** [comment]
@chiarazampolli 
PR connected to the following ones:

- QualityControl: https://github.com/AliceO2Group/QualityControl/pull/2021
- O2DPG: https://github.com/AliceO2Group/O2DPG/pull/1287

**Review by chiarazampolli — DISMISSED** [review]

**Comment by mfaggin** [comment]
hi @shahor02 @chiarazampolli  can this PR go ahead?

File: Detectors/GlobalTracking/include/GlobalTracking/TrackCuts.h [hunk 1]
```diff
@@ -60,62 +86,68 @@ class TrackCuts
...
-    if (isBarrelTrack) { // track selection for barrel tracks
+    // ITS-TPC matched cuts
+    // --> currently inactive in MatchITSTPCQC, since either GID::TPC or GID::ITS
+    if (src == o2::dataformats::GlobalTrackID::ITSTPC ||
+        src == o2::dataformats::GlobalTrackID::ITSTPCTRD ||
+        src == o2::dataformats::GlobalTrackID::ITSTPCTOF ||
+        src == o2::dataformats::GlobalTrackID::ITSTPCTRDTOF) { // track selection for barrel tracks (ITS-TPC matched)
       trk = data.getTrackParam(trackIndex);
-      if (trk.getPt() < mMinPt && trk.getPt() > mMaxPt && trk.getEta() < mMinEta && trk.getEta() > mMaxEta) {
+      if (trk.getPt() < mMinPt || trk.getPt() > mMaxPt || trk.getEta() < mMinEta || trk.getEta() > mMaxEta) {
```
Comment (shahor02) [inline Detectors/GlobalTracking/include/GlobalTracking/TrackCuts.h]: getEta is not a getter but does quite heavy calculation. If it is used more than once better assign it to local variable and reuse it.

**Review by shahor02 — COMMENTED** [review]
@mfaggin  please see the comment below. If you want, this can go to separate pr.

**Comment by mfaggin** [comment]
> @mfaggin please see the comment below. If you want, this can go to separate pr.

Hi @shahor02 , I've just done it

**Review by shahor02 — APPROVED** [review]
Thanks! Merging since a trivial change was applied over the green CI.

**Pull request merged**